### ISSUE TITLE
There are text elements in the solution that have a poor colour contrast ratio against their backgrounds when the page is viewed in Windows High Contrast mode.

### ISSUE THREAD

**Issue by adorayi** [cds-snc/design-gc-conception#2113]
WCAG requires that there be a contrast ratio of at least 3:1 for large text and 4.5:1 for non-large text.  
Example 1: The text ""Canada"" found in the logo on the bottom right of the page has a text colour of #000000 and background colour of #202020 in Window High Contrast mode resulting in a colour contrast of 1.29:1.  

**Impact on Users:**   

Some users with visual impairments may experience difficulties seeing text with a poor colour contrast against its background.

We recommend adjusting the foreground and/or background colours of all affected text elements until they meet a contrast ratio of at least 3:1 for large text and 4.5:1 for non-large text.

**Additional Resources:**

https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

**Steps to Reproduce:**

1. Open Settings on your Windows PC and search for "Contrast theme" then select the "Choose Contrast theme" option.
2. Select a Dark contrast theme and Apply it to your Windows PC.
3. Open the solution: https://design-system.alpha.canada.ca/en/components/
4. Take a screenshot of one of the elements in the examples identified in the description and identify its foreground and background colours. 
5. Plug the colours into a colour contrast checker such as: https://coolors.co/contrast-checker

**Actual Result:** 

The colour contrast does not meet the minimum set by WCAG guidelines.

**Expected Result:** 

The colour contrast meets the minimum set by WCAG guidelines.

**Comment by ethanWallace** [comment]
Added CSS to handle the use of high contrast themes. The FIP elements will swap between the black and white versions depending if the theme is light or dark.
